anthos-baremetal-edge-deployment/roles/abm-post-install/templates/collectd.conf.j2

```diff
@@ -0,0 +1,141 @@
+Interval 60
+
+# Explicitly set hostname to "" to indicate the default resource.
+Hostname "{{ machine_label }}"
+
+# The Stackdriver agent does not use fully qualified domain names.
+FQDNLookup false
+
+LoadPlugin syslog
+<Plugin "syslog">
+  LogLevel "info"
+</Plugin>
+# if you uncomment this, you will get collectd logs separate from syslog
+LoadPlugin logfile
+<Plugin "logfile">
+  LogLevel "info"
+  File "/var/log/collectd.log"
+  Timestamp true
+</Plugin>
+
+LoadPlugin df
+<Plugin "df">
+  FSType "devfs"
+  IgnoreSelected true
+  ReportByDevice true
+  ValuesPercentage true
+</Plugin>
+
+LoadPlugin cpu
+<Plugin "cpu">
+  ValuesPercentage true
+  ReportByCpu false
+</Plugin>
+LoadPlugin swap
+<Plugin "swap">
+  ValuesPercentage true
+</Plugin>
+LoadPlugin interface
+LoadPlugin disk
+LoadPlugin load
+LoadPlugin memory
+<Plugin "memory">
+  ValuesPercentage true
+</Plugin>
+LoadPlugin processes
+LoadPlugin tcpconns
+LoadPlugin write_gcm
+LoadPlugin match_regex
+LoadPlugin match_throttle_metadata_keys
+LoadPlugin stackdriver_agent
+
+<Plugin "interface">
+  Interface "eno1"
+  Interface "eno1.410"
```

Comment:
You can remove this line and the following (460)...these are specific to VLANs that won't be used

Reply:
Removed in f7a7516. Confirm it is correct please!